packages/expo-updates/ios/EXUpdates/AppController.swift

```diff
@@ -463,11 +463,14 @@ public class AppController: NSObject, AppLoaderTaskDelegate, ErrorRecoveryDelega
         updateId: update.loggingId(),
         assetId: nil
       )
-      stateMachine?.processEvent(UpdatesStateEventDownloadCompleteWithUpdate(manifest: update.manifest!.rawManifestJSON()))
-      // Send UpdateEvents to JS
-      sendLegacyUpdateEventToBridge(AppController.UpdateAvailableEventName, body: [
-        "manifest": update.manifest!.rawManifestJSON()
-      ])
+      // Ensure manifest is non-null before sending events
+      if let manifest = update.manifest {
+        stateMachine?.processEvent(UpdatesStateEventDownloadCompleteWithUpdate(manifest: manifest.rawManifestJSON()))
+        // Send UpdateEvents to JS
+          sendLegacyUpdateEventToBridge(AppController.UpdateAvailableEventName, body: [
+            "manifest": manifest.rawManifestJSON()
+          ])
```

Comment:
nit: indentation

Reply:
Hmm this passed Swiftlint, but I will take a look

Reply:
Fixed